### PR TITLE
Implementation Stage-1

### PR DESCRIPTION
**Pull request by PanRadaCloudbest**
## What does this PR do?

This PR add:
- seed confirm flow
- factory reset in settings

Fix:
- flow import wallets

## Todos:

- [x] Checked on iOS
- [x] Checked on Android

### CODE REVIEW

File: loc/es.js [hunk 1]
```diff
@@ -485,6 +487,13 @@ module.exports = {
     notSupportedFingerPrint: 'Su dispositivo no es compatible con la huella dactilar ',
     security: 'Seguridad ',
     TouchID: 'Permitir huella digital ',
+    factory: {
+      title: 'Restablecer la configuración de fábrica',
+      text:
+        '¿Está seguro de que quiere restaurar Gold Wallet a su configuración de fábrica? Todos tus monederos y datos serán borrados. Recuerde que tendrá que importar sus monederos de nuevo.',
+      noButton: 'No ',
```
Comment (aleparty) [inline loc/es.js:494]: excessive spaces

File: loc/es.js [hunk 2]
```diff
@@ -485,6 +487,13 @@ module.exports = {
     notSupportedFingerPrint: 'Su dispositivo no es compatible con la huella dactilar ',
     security: 'Seguridad ',
     TouchID: 'Permitir huella digital ',
+    factory: {
+      title: 'Restablecer la configuración de fábrica',
+      text:
+        '¿Está seguro de que quiere restaurar Gold Wallet a su configuración de fábrica? Todos tus monederos y datos serán borrados. Recuerde que tendrá que importar sus monederos de nuevo.',
+      noButton: 'No ',
+      yesButton: 'Sí ',
```
Comment (aleparty) [inline loc/es.js:495]: excessive spaces

**Review by dgabka — APPROVED** [review]

**Review by aleparty — APPROVED** [review]
Cosmetic changes

File: loc/es.js (hunk 1 above)
Comment (PanRadaCloudbest) [reply loc/es.js]: done

File: loc/es.js (hunk 2 above)
Comment (PanRadaCloudbest) [reply loc/es.js]: done

**Review by aleparty — APPROVED** [review]

**Pull request merged**